src/gui/application.cpp

```diff
@@ -336,7 +344,7 @@ void Application::slotownCloudWizardDone(int res)
 #ifdef Q_OS_MAC
         // Don't auto start when not being 'installed'
         shouldSetAutoStart = shouldSetAutoStart
-            && QCoreApplication::applicationDirPath().startsWith("/Applications/");
+                             && QCoreApplication::applicationDirPath().startsWith("/Applications/");
 #endif
         Utility::setLaunchOnStartup(_theme->appName(), _theme->appNameGUI(), shouldSetAutoStart);
```

Comment:
Did you see this invocation?

Reply:
❓ ❔ 😕

Reply:
obviously not ..... crawling back into my cave

Reply:
Note that this DISABLE the auto start when setting up a second account.

Reply:
So if I get this right: as soon as one account is setup auto launch is active.
why is it disabled with the second account ? THX

Reply:
maybe that's a bug.  @guruz  why do you disable auto-launch with multi account

Reply:
It looks like a mistake: the commit message of fc7aaf792b26c22adaef6997bb18c58fe2c18f94 suggests it was intended to not adjust the autostart flag when further accounts are added.